docs/introduction/welcome.mdx

```diff
@@ -1,22 +1,31 @@
 ---
-title: Welcome to BeeAI
+title: Welcome
 sidebarTitle: Welcome
-description: "Discover, run, and compose AI agents from any framework"
+description: "Discover, run, and orchestrate AI agents from any framework"
 icon: "door-open"
 ---
 
 ## What is BeeAI?
 
-BeeAI is an open platform to help you discover, run, and compose AI agents from any framework and language. Whether building your agents or looking for powerful existing solutions, BeeAI makes it easy to find, connect, and orchestrate AI agents seamlessly.
+**BeeAI** is the first open-source platform built on the [Agent Communication Protocol (ACP)](https://agentcommunicationprotocol.dev/), an open standard designed to enable smooth communication between AI agents, no matter what framework they're built on. Hosted by the **Linux Foundation**, BeeAI is driven by open governance and community collaboration.
+
+With ACP at its core, BeeAI makes it easy to:
+- **Discover** agents across a wide range of frameworks via our growing catalog  
+- **Run** agents locally using your preferred LLM provider  
```

Comment:
I think we should emphasize the "try experience" 

firstly, we have really nice GUI & CLI UI and secondly that's one of the common pain points in the community that if they want to try some of open source agent, it's often just painful to install.